<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window2</class>
 <widget class="QMainWindow" name="window2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="window2_check_btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>check</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="window2_result_field">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>451</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="window2_input_field">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
